tools/compat/modules/import_meta.md

```diff
@@ -0,0 +1,283 @@
+---
+__compat:
+  description: "<code>import.meta</code>"
+  mdn_url: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/import.meta
+  spec_url:
+  - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import.meta
+  - https://vite.dev/guide/env-and-mode
+  - https://webpack.js.org/api/module-variables/#importmeta
```

Comment:
Does this yaml schema support adding these as links specific to the tool, rather than in general? Like a per-tool note.